### PR TITLE
Replicate aim

### PR DESCRIPTION
**Pull request by Sorseg**
https://github.com/Sorseg/petrichor4/assets/478878/44ee6b91-bc71-4866-b9c1-6e67e2447d21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ Run client
 ```shell
 cargo run --bin petri_client --features bevy/dynamic_linking
 ```
+
+## Bevy coordinates
+
+- The X axis goes from left to right (+X points right).
+- The Y axis goes from bottom to top (+Y points up).
+- The Z axis goes from far to near (+Z points towards you, out of the screen).
```
Comment (demoth) [inline README.md:19]:
```suggestion
- The Z axis goes from far to near (+Z points towards you, -Z out of the screen).
```

File: crates/petri_client/src/plugin.rs [hunk 2]
```diff
@@ -145,6 +153,7 @@ impl Plugin for PetriClientPlugin {
             // FIXME: limit X turn angle
             transform.rotate_y(-delta.x * sensitivity);
             transform.rotate_local_x(-delta.y * sensitivity);
+            events.send(Aim(transform.forward()));
```
Comment (demoth) [inline crates/petri_client/src/plugin.rs:156]: do we send yaw? or pitch also?

File: crates/petri_server/src/plugin.rs [hunk 3]
```diff
@@ -84,36 +86,42 @@ impl Plugin for PetriServerPlugin {
...
-                                linear_damping: 2.0,
-                                angular_damping: 0.0,
-                            },
-                        ));
+                                PhysicsBundle {
+                                    collider: Collider::capsule_y(
+                                        capsule_segment_half_height,
+                                        capsule_diameter / 2.0,
+                                    ),
+                                    trans: TransformBundle::from_transform(Transform::from_xyz(
```
Comment (demoth) [inline crates/petri_server/src/plugin.rs:101]: // todo: replace with a `info_player_spawn` entity

File: crates/petri_client/src/plugin.rs (hunk 2 above)
Comment (Sorseg) [reply crates/petri_client/src/plugin.rs:156]: This is a `Direction3D`, which is a 3D vector with no rotational information around that axis

**Review by demoth — APPROVED** [review]

**Pull request merged**